platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/TelemetryDefinition.java

```diff
@@ -46,4 +48,10 @@
    * @param offlineDefinition the offline region definition
    */
   void onCreateOfflineRegion(OfflineRegionDefinition offlineDefinition);
+
+  /**
+   * Register performance event.
+   * @param data performance event data
+   */
+  void onPerformanceEvent(Bundle data);
```

Comment:
While we are introducing a performance event that can be reused for multiple purposes, I would prefer keeping the  TelemetryDefinition interface to be as concrete as possible. Some users swap out our telemetry with their own and wouldn't know how to implement `onPerformanceEvent(Bundle data)`. Having a method with tile latency and adding a long duration parameter would make it more clear and easier for those users to integrate.